app/main.py

```diff
@@ -26,14 +26,27 @@ def upload():
         return redirect(url_for('index'))
 
     file = request.files['file']
-    filename = file.filename
+    filename = file.filename    
+
     # if user does not select file, browser submits an empty part without filename
     if filename == '':
         flash('Please select a CSV file to Upload!', 'error')
         return redirect(url_for('index'))
 
+    # If a PNG is uploaded, push it to the folder
+    if request.files['image'].filename != '':
```

Comment:
Here it is image.

Reply:
Hi Nicco, The 'imageFile' is the ID, but the request data is accessed using the "name" attribute.
Fixing conflicts in a minute